libasn1fix/asn1fix_cws.c

```diff
@@ -307,6 +307,8 @@ _asn1f_assign_cell_value(arg_t *arg, struct asn1p_ioc_row_s *row, struct asn1p_i
 			FATAL("Cannot find %s referenced by %s at line %d",
 				p, arg->expr->Identifier,
 				arg->expr->_lineno);
+			asn1p_ref_free(ref);
```

Comment:
This pull request and previous #23 are both based on [651779...](https://github.com/mouse07410/asn1c/commit/651779490828d4ef45b081a4699e40a53aa97fc8). I tried to make these two pull requests can be accepted or rejected independently. 

However, because re-written of `_asn1f_assign_cell_value()`, so the modified code segments are conflicted if both pull requests accepted.